lib/vc.js

```diff
@@ -310,13 +310,20 @@ async function _verifyPresentation(options = {}) {
   };
 }
 
+/**
+ * @param {String|Object} obj
+ * @returns {String|undefined}
+ * @private
+ */
 function _getId(obj) {
   if(typeof obj === 'string') {
     return obj;
   }
+
   if(!('id' in obj)) {
-    throw new Error('"id" not found.');
+    return undefined;
```

Comment:
First, when returning `undefined` just do `return`. But... this will break the issuer check. `issuer` still needs to have an `id` if it's an object.

Reply:
I added a separate existence check for the issuer, in that section.

Reply:
(Removed 'return undefined')